validation-test/compiler_crashers_fixed/28638-anonymous-namespace-verifier-checksametype-swift-type-swift-type-char-const.swift

```diff
@@ -5,5 +5,5 @@
 // See https://swift.org/LICENSE.txt for license information
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 
-// RUN: %target-swift-frontend %s -emit-ir
-nil as Int!as?Int?!
+// RUN: %target-swift-frontend %s -emit-ir -verify
```

Comment:
Change this to 'not %target-swift-frontend %s -typecheck', and drop the -verify. We don't want validation tests to verify diagnostics

Reply:
Yes, good point.